src/Go/boardAnalysis/boardAnalysis.ts

```diff
@@ -1,6 +1,6 @@
-import type { Board, BoardState, Neighbor, PointState, SimpleBoard } from "../Types";
+import { Board, BoardState, Neighbor, Play, PointState, SimpleBoard } from "../Types";
```

Comment:
Although technically the way we have things configured right now there's no issues with importing stuff that we use only for types, it's a good habit. At some point (maybe) we might enable some options to get the codebase clean in that regard.

To that effect: I'd leave the existing ones as `import type`, and and new line for the regular import.

Reply:
This one was my IDE changing it, it definitely should all be import type